Entities/AtomGroup.cpp

```diff
@@ -65,22 +65,21 @@ namespace RTE {
 				atomCopy->SetIgnoreMOIDsByGroup(&m_IgnoreMOIDs);
 				m_Atoms.push_back(atomCopy);
 
-				// Add to the appropriate spot in the subgroup map
 				long subgroupID = atomCopy->GetSubID();
 				if (subgroupID != 0) {
-					// Make a new list for the subgroup ID if there isn't one already
 					if (m_SubGroups.find(subgroupID) == m_SubGroups.end()) { m_SubGroups.insert({ subgroupID, std::list<Atom *>() }); }
 
 					m_SubGroups.find(subgroupID)->second.push_back(atomCopy);
 				}
 			}
 		}
 
+		m_IgnoreMOIDs.clear();
```

Comment:
What's up with clearing IgnoreMOIDs here? Maybe we talked about this and I've forgotten or something?